helix-term/src/commands/lsp.rs

```diff
@@ -134,7 +139,7 @@ struct DiagnosticStyles {
 }
 
 struct PickerDiagnostic {
-    url: lsp::Url,
+    uri: helix_core::Uri,
```

Comment:
can we just import `Uri`? Ithink keeping it as a path makes this more verbose without making it more clear